pkg/scheduler/api/node_info.go

```diff
@@ -193,3 +220,11 @@ func (ni *NodeInfo) Pods() (pods []*v1.Pod) {
 
 	return
 }
+
+// GetAccessibleResource get accessible resource which equals idle + backfilled
+func (ni *NodeInfo) GetAccessibleResource() *Resource {
+	idle := ni.Idle.Clone()
+	accessible := idle.Add(ni.Backfilled).Clone()
```

Comment:
It's not necessary to clone again, right?

Reply:
fixed